webapp/constants/manifest.js

```diff
@@ -1,9 +1,10 @@
 import metadata from './metadata.js'
-const { APPLICATION_NAME, APPLICATION_SHORT_NAME, APPLICATION_DESCRIPTION } = metadata
+const { APPLICATION_NAME, APPLICATION_SHORT_NAME, APPLICATION_DESCRIPTION, THEME_COLOR } = metadata
+
 export default {
   name: APPLICATION_NAME,
   short_name: APPLICATION_SHORT_NAME,
   description: APPLICATION_DESCRIPTION,
-  theme_color: '#17b53f',
+  theme_color: THEME_COLOR,
```

Comment:
- using a new SCSS name like `'$theme-color'` doesn't work here unfortunately
- to import the color from SCSS gives a nuxt error unfortunately